### PR TITLE
zh-TW

### PR DESCRIPTION
**Pull request by silverbow1**
chinese traditional -Taiwan

### CODE REVIEW

File: locales/zh-TW.js [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+confirm_order:
+"確認訂單",
+bill_me_later:
+"一會兒再結帳",
+bill_me_later_action:
+"請稍後為您撥打電話",
+bill_me_later_explanation:
+"發票將電子郵件發送給您",
+pay_via_mollie:
+"選擇付款方式",
```
Comment (JeanSebTr) [inline locales/zh-TW.js]: [Mollie](https://www.mollie.com/) is a brand, I would expect it to look a bit like Paypal. Could you validate? 🙂

File: locales/zh-TW.js [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+payment_informations_bill_me_later:
+"我稍後會付款",
+payment_informations_no_payment_required:
+"無須付款",
+payment_informations_paypalexpress:
+"用Paypal付款",
+credit_card_type_mastercard:
+"萬事通",
+credit_card_type_visa:
+"信用卡",
```
Comment (JeanSebTr) [inline locales/zh-TW.js]: Google Translate tells me "信用卡" simply means "credit card". Is it the way Visa is referred to?

File: locales/zh-TW.js (hunk 1 above)
Comment (silverbow1) [reply locales/zh-TW.js]: Here, in Taiwan, they have a name for it. If you prefer, you can change it but not sure taiwanese will understand

File: locales/zh-TW.js (hunk 2 above)
Comment (silverbow1) [reply locales/zh-TW.js]: google translate is wrong. it is simple chinese and not correct.

**Comment by JeanSebTr** [comment]
Thanks @silverbow1 for the confirmations!

Merging it now and it will be available in the cart later this week.

**Pull request merged**